local.css

```diff
@@ -46,94 +51,101 @@ blockquote{
 	padding: 10px;
 }
 
-
 /**** Main Container ***/
 
 
-#wrap {
-       margin-left: auto;
-       margin-right: auto;
-       width: 730px;
+#wrap {     
+       width: 100%;
+       height: 100%;
 }
 
 
 /**** Top ***/
 
 
 #top {
-	font-size: 0.8em;
+	font-size: 1.0em;
 	width: 100%;
-	height: 88px;
-	color: #fff;
-	/*background: #000 url(images/top_bg.gif);*/
+	margin-left: 0;
 	overflow:hidden;
+	display: inline-block;
 }
 
 #top h2 {
-	color: White;
 	letter-spacing: 3px;
-	font-size: 2.0em;
+	font-size: 3.0em;
 	font-weight: bold;
 	position: relative;
-	margin: 0px;
-	top:25px;
+	/* margin: 0px; */
+	top: 5px;
+	margin-right: 20px;
 	display:block;
 	float:left;
-	background: url(images/bg_t.gif) no-repeat;
-	padding-left: 10px;
 }
 
-#top h2 a {
-	color: white;
+#top h2 a {	
 	text-decoration: none;
 }
 
 #top h2 a:hover {
-        color: #347272;
+	color: black;        
+}
+
+/**** Center Logo ****/
+
```

Comment:
Not sure about the "header" itself, but it should at least have no newline below.
What about Vim markers.. '{{{` … `}}}`